src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final int index = new Random().nextInt(Color.values().length);
+
+    public String getRandomColor() {
+        return Color.values()[index].toString().toLowerCase();

```

Comment:
It’s better to use standard method of enum name() that will be returning always String representation of concrete enum constant. Don’t use toString() or String.valueOf()(it will call toString() under the hood) for getting String representation of enum constants. toString() is common for all enum constants.